### PR TITLE
Adapting rust code to dex-contract PR with removed IsSellOrderFlag

### PR DESCRIPTION
**Pull request by josojo**
as title

---
testplan:
e2e tests & unit tests

### CODE REVIEW

File: docker/truffle/Dockerfile [hunk 1]
```diff
@@ -7,5 +7,4 @@ COPY docker/truffle/run.sh /app/run.sh
 
 # Dependencies will Install app dependencies
 RUN apk add --update --no-cache --virtual build-dependencies git python make g++ ca-certificates
-RUN npm install --production
```
Comment (josojo) [inline docker/truffle/Dockerfile]: I will put this into a separate PR, as it is an fix

File: driver/src/contracts/stablex_auction_element.rs [hunk 2]
```diff
@@ -19,7 +18,7 @@ impl StableXAuctionElement {
         self.valid_from <= index && index <= self.valid_until
     }
 
-    pub fn from_bytes(order_count: &mut HashMap<H160, u16>, bytes: &[u8; 113]) -> Self {
+    pub fn from_bytes(order_count: &mut HashMap<H160, u16>, bytes: &[u8; 112]) -> Self {
```
Comment (fleupold) [inline driver/src/contracts/stablex_auction_element.rs]: Should this use the const `AUCTION_ELEMENT_WIDTH`?

File: docker/truffle/Dockerfile (hunk 1 above)
Comment (fleupold) [reply docker/truffle/Dockerfile]: Check #280 for a fix.

**Comment by bh2smith** [comment]
Any idea why I can't see the little ❌s and ✅s  beside the commits? 

Screen Shot 2019-10-31 at 8 55 43 AM

**Comment by bh2smith** [comment]
The travis has not detected the last three commits here.

**Pull request merged**